app/src/main/res/layout/adapter_kind_items.xml

```diff
@@ -14,56 +14,78 @@
-            android:layout_marginStart="8dp"
-            android:layout_marginEnd="8dp"
             android:background="@drawable/placeholder_stalls"
-            app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintDimensionRatio="1.33"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent" />
 
     <TextView
             android:id="@+id/ItemName"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
+            android:layout_width="225dp"
+            android:layout_height="36dp"
+            android:layout_marginStart="51dp"
+            android:layout_marginTop="16dp"
+            android:layout_marginEnd="54dp"
+            android:layout_marginBottom="56dp"
+            android:ellipsize="end"
             android:fontFamily="@font/shellypalmer_circular_std_book"
+            android:maxLines="1"
+            android:scrollHorizontally="true"
             android:text="Stall Name"
             android:textAlignment="center"
             android:textColor="@color/colorWhite"
-            android:textSize="24sp"
+            android:textSize="22sp"
             android:textStyle="bold"
-            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintBottom_toBottomOf="@+id/stallImage"
             app:layout_constraintEnd_toEndOf="@+id/stallImage"
-            app:layout_constraintStart_toStartOf="@+id/stallImage"
+            app:layout_constraintStart_toEndOf="@+id/kImg"
             app:layout_constraintTop_toTopOf="@+id/stallImage" />
 
     <TextView
             android:id="@+id/price"
-            android:layout_width="30dp"
-            android:layout_height="0dp"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="169dp"
+            android:layout_marginTop="56dp"
+            android:layout_marginBottom="16dp"
             android:fontFamily="@font/shellypalmer_circular_std_book"
             android:gravity="center"
             android:text="0"
             android:textAlignment="center"
             android:textColor="@color/colorWhite"
-            android:textSize="24sp"
-            android:textStyle="bold"
-            app:layout_constraintBottom_toBottomOf="@+id/ItemName"
-            app:layout_constraintEnd_toEndOf="@+id/stallImage"
-            app:layout_constraintHorizontal_bias="0.6"
-            app:layout_constraintStart_toEndOf="@+id/ItemName"
-            app:layout_constraintTop_toTopOf="@+id/ItemName" />
+            android:textSize="22sp"
+            app:layout_constraintBottom_toBottomOf="@+id/stallImage"
+            app:layout_constraintStart_toStartOf="@+id/stallImage"
+            app:layout_constraintTop_toTopOf="@+id/stallImage" />
 
     <ImageView
             android:id="@+id/kImg"
```

Comment:
Also, change the name of this image to a more meaningful one

Reply:
Okay